src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -116,6 +120,57 @@ const RenderListFeed = (props) => {
     initialSetup(item);
     checkVotesHandle();
   }, [item]);
+
+  const hasComment = getCommentLength(item.latest_reactions.comment) > 0;
+
+  const isBlurred = item?.isBlurredPost && item?.anonimity;
+
+  const renderWriteComment = () => {
+    return (
+      !isBlurred && (
+        <TouchableOpacity
+          testID="writeComment"
+          onPress={() => onPressComment(isHaveSeeMore)}
+          style={styles.contentReaction(getHeightReaction())}>
+          <WriteComment
+            postId={''}
+            username={handleUserName(item)}
+            value={''}
+            onChangeText={() => {}}
+            onPress={() => {}}
+            loadingPost={false}
+            isViewOnly={true}
+            withAnonymityLabel={false}
+          />
+        </TouchableOpacity>
+      )
+    );
+  };
+
+  const commentHeight = () => {
+    if (isBlurred && !hasComment) {
+      return 0;
+    }
+    const isSingleComment = getTotalReaction(item) === 1;
+    const commentSectionHeight = getHeightReaction() - getHeightFooter();
+    return isSingleComment ? commentSectionHeight - 20 : commentSectionHeight;
+  };
+
+  const {onLayoutTopicChip} = useCalculationContent();
+  const calculateLineTopicChip = (nativeEvent) => {
+    onLayoutTopicChip(nativeEvent, 1);
+  };
+
+  const topicBottomPosition = () => {
+    if (hasComment) {
+      if (getTotalReaction(item) === 1) {
+        return getHeightReaction() - 20;
+      }
+      return getHeightReaction();
+    }
+    return getHeightFooter();
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for checking if a post has comments and whether it should be blurred is clear and well implemented. However, the `renderWriteComment` function seems to be missing some necessary props for the `WriteComment` component such as `postId` and `onChangeText`. These are currently hardcoded with empty strings or no-op functions which might lead to unexpected behavior.
```diff
-            postId={''}
-            onChangeText={() => {}}
+            postId={item.id}
+            onChangeText={handleTextChange}
```
Also, the `onPress` prop is currently a no-op function. If there's no need for an action on press, you can remove this prop.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this component is just for view only